package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "specification",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
Does this bump a whole minor version? Why not patch? (Or should the previous changes themselves also have been minor bumps?)